robyn/__main__.py

```diff
@@ -0,0 +1,39 @@
+import click
+import os
+from robyn import Robyn
```

Comment:
Also, we are not using this statement. We can remove this 😄

Reply:
@sansyrox 
I tested this and this approach don't seems to fix the issue
so....that means we're not using click again??

Reply:
@sansyrox All done you can now check the improved version